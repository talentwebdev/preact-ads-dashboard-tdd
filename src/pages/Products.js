import React from 'react'

function Products() {
    return (
        <div>
            This is a product page
        </div>
    )
}

export default Products
