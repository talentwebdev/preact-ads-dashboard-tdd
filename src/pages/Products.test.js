import {shallow} from 'enzyme';
import Products from './Products';

describe('Product', () => {
    let productsContainer;

    beforeEach(() => productsContainer = shallow(<Products />));

    it('should render the div element', () => {
        expect(productsContainer.find('div')).toBeTruthy();
    });
});