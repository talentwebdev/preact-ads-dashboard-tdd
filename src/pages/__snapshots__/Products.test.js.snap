// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product renders Products correctly 1`] = `ShallowWrapper {}`;
